### DISCUSSION TITLE
@antmjs/vantui 基于有赞微信小程序改造的可以同时在Taro3-React和React中使用的组件库

### DISCUSSION THREAD

**Discussion by hisanshao** [NervJS/taro#10586] [生态]
1: 我们认为它可以为Vant Weapp以及Taro锦上添花[文档查看](https://antmjs.github.io/vantui/#/home)
2: 我们会持续同步Vant Weapp小程序的最新版，目前同步至最新的1.9.2
3: 希望可以在Taro社区生态内加入我们的仓库，公司内部已经在推广使用中

**Comment by hisanshao** [comment]
物料提交不上去，报错undefined

**Reply by honlyHuang** [reply]
> 物料提交不上去，报错undefined

加我微信（微信号: honly_huang）咯，截图我看看的
